pkg/secrets/fetch_secret_darwin.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Why duplicate the linux code for darwin ? We could just have a windows version and a non windows version.